NullReferenceException in Android TabbedPageManager

### Description

```
[AppCenterXamarinCrashes] Unhandled Exception from source=AppDomain
[AppCenterXamarinCrashes] System.NullReferenceException: Object reference not set to an instance of an object.
[AppCenterXamarinCrashes]    at Microsoft.Maui.Controls.Handlers.TabbedPageManager.Listeners.OnPageSelected(Int32 position) in D:\a\_work\1\s\src\Controls\src\Core\Platform\Android\TabbedPageManager.cs:line 843
[AppCenterXamarinCrashes]    at AndroidX.ViewPager2.Widget.ViewPager2.OnPageChangeCallback.n_OnPageSelected_I(IntPtr jnienv, IntPtr native__this, Int32 position) in C:\a\_work\1\s\generated\androidx.viewpager2.viewpager2\obj\Release
et6.0-android\generated\src\AndroidX.ViewPager2.Widget.ViewPager2.cs:line 318
[AppCenterXamarinCrashes]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPI_V(_JniMarshal_PPI_V callback, IntPtr jnienv, IntPtr klazz, Int32 p0) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 59
[mono-rt] [ERROR] FATAL UNHANDLED EXCEPTION: System.NullReferenceException: Object reference not set to an instance of an object.
[mono-rt]    at Microsoft.Maui.Controls.Handlers.TabbedPageManager.Listeners.OnPageSelected(Int32 position) in D:\a\_work\1\s\src\Controls\src\Core\Platform\Android\TabbedPageManager.cs:line 843
[mono-rt]    at AndroidX.ViewPager2.Widget.ViewPager2.OnPageChangeCallback.n_OnPageSelected_I(IntPtr jnienv, IntPtr native__this, Int32 position) in C:\a\_work\1\s\generated\androidx.viewpager2.viewpager2\obj\Release
et6.0-android\generated\src\AndroidX.ViewPager2.Widget.ViewPager2.cs:line 318
[mono-rt]    at Android.Runtime.JNINativeWrapper.Wrap_JniMarshal_PPI_V(_JniMarshal_PPI_V callback, IntPtr jnienv, IntPtr klazz, Int32 p0) in /Users/runner/work/1/s/xamarin-android/src/Mono.Android/Android.Runtime/JNINativeWrapper.g.cs:line 59
```

Randomly happens upon navigating BACK from a page, our app consist of a `NavigationPage` holding a page that extends `Microsoft.Maui.Controls.TabbedPage` .

I found #10282 which seems to be this exact issue. Again, it happens sporadically.

### Steps to Reproduce


``` 
    [XamlCompilation(XamlCompilationOptions.Compile)]
    public partial class AssignmentTabbedContainer : Microsoft.Maui.Controls.TabbedPage
    {
        public bool IsFirstLoad { get; set; }
        public bool isAttachmentPressed { get; set; }
        public AssignmentTabbedContainer()
        {
            InitializeComponent();
        }
    }
``` 

```
<?xml version="1.0" encoding="utf-8" ?>
<TabbedPage x:Class="Signco.Mobile.MaaS.Maui.Pages.AssignmentTabbedContainer.AssignmentTabbedContainer"
            xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
            xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
            xmlns:shared="clr-namespace:Signco.Mobile.MaaS.Maui.Shared"
            xmlns:android="clr-namespace:Microsoft.Maui.Controls.PlatformConfiguration.AndroidSpecific;assembly=Microsoft.Maui.Controls"
            android:TabbedPage.ToolbarPlacement="Bottom"
            x:DataType="viewModels:TabbedPageViewModel"
            ItemsSource="{Binding Tabs}"
            SelectedItem="{Binding SelectedTab, Mode=TwoWay}">

    <NavigationPage.TitleView>
        <shared:TitleView Title="{Binding Title}" />
    </NavigationPage.TitleView>

    <TabbedPage.ItemTemplate>
        <DataTemplate x:DataType="viewModels:TabContentPageViewModel">
            <ContentPage Title="{Binding Title}"
                         Shell.NavBarIsVisible="false"
                         Content="{Binding Content.Content}"
                         IconImageSource="{Binding Icon}" />
        </DataTemplate>
    </TabbedPage.ItemTemplate>
</TabbedPage>
```

Navigating "back" can cause this error to happen.

### Link to public reproduction project repository

_No response_

### Version with bug

8.0.10 SR3

### Is this a regression from previous behavior?

Yes, this used to work in Xamarin.Forms

### Last version that worked well

Unknown/Other

### Affected platforms

Android

### Affected platform versions

_No response_

### Did you find any workaround?

_No response_

### Relevant log output

_No response_

--- a/src/Controls/src/Core/Platform/Android/TabbedPageManager.cs
+++ b/src/Controls/src/Core/Platform/Android/TabbedPageManager.cs
@@ -899,8 +899,12 @@ namespace Microsoft.Maui.Controls.Handlers
 			{
 				base.OnPageSelected(position);
 
-				var _previousPage = _tabbedPageManager._previousPage;
 				var Element = _tabbedPageManager.Element;
+
+				if (Element == null)
+					return;
+
+				var _previousPage = _tabbedPageManager._previousPage;
 				var IsBottomTabPlacement = _tabbedPageManager.IsBottomTabPlacement;
 				var _bottomNavigationView = _tabbedPageManager._bottomNavigationView;
 

